README.md

```diff
@@ -1 +1,106 @@
+    PS> .\vcpkg install libpng sdl1 sdl1-net opusfile
+
+These two steps will ensure that MSVC finds and links all dependencies.
+
+Start Visual Studio, open file: `visualc_net\dosbox.sln` and build all
+projects (Ctrl+Shift+B).
+
+
+## Interop with SVN
+
+This repository is (deliberately) NOT git-svn compatible, this is a pure
+Git repo.
+
+Commits landing in SVN upstream are imported to this repo in a timely manner,
+to the branches matching `svn/*` pattern, e.g.
+[`svn/trunk`](https://github.com/dreamer/dosbox-staging/tree/svn/trunk).
+You can safely use those branches to rebase your changes, and prepare patches
+using Git [format-patch](https://git-scm.com/docs/git-format-patch) for sending
+upstream (it is easier and faster, than preparing patches manually).
+
+Git tags matching pattern `svn/*` are pointing to the commits referenced by SVN
+"tag" paths at the time of creation.
+
+Additionally, we attach some optional metadata to the commits imported from SVN
+in the form of [Git notes](https://git-scm.com/docs/git-notes). To fetch them,
+run:
+
+    $ git fetch origin "refs/notes/*:refs/notes/*"
+
+
+## Summary of differences compared to upstream:
+
+|                                | dosbox-staging              | DOSBox
+|-                               |-                            |-
+| **Version control**            | Git                         | [SVN]
+| **Language**                   | C++11                       | C++03<sup>[1]</sup>
+| **CI**                         | Yes                         | No
+| **Static analysis**            | Yes                         | No
+| **Dynamic analysis**           | Yes                         | No
+| **Automated regression tests** | No (planned)<sup>[2]</sup>  | No
+| **SDL**                        | 1.2 (2.0 WIP)<sup>[3]</sup> | 1.2
+
+[SVN]:https://sourceforge.net/projects/dosbox/
+[1]:https://sourceforge.net/p/dosbox/patches/283/
+[2]:https://github.com/dreamer/dosbox-staging/issues/23
+[3]:https://github.com/dreamer/dosbox-staging/issues/29
+
+Codecs supported for CD Digital Audio emulation (loading CD music via
+[cue sheets](https://en.wikipedia.org/wiki/Cue_sheet_(computing))):
+
+|                       | dosbox-staging | DOSBox
+|-                      |-               |-
+| **Opus**              | libopus        | No
+| **FLAC**              | built-in       | SDL\_sound 1.2 (libflac)<sup>†</sup>
+| **OGG/Vorbis**        | built-in       | SDL\_sound 1.2 (libvorbis)<sup>†</sup>
+| **MP3**               | built-in       | SDL\_sound 1.2 (libmpg123)<sup>†</sup>
+| **WAV**               | built-in       | SDL\_sound 1.2 (internal)<sup>†</sup>
+| **Speex**<sup>‡</sup> | No             | SDL\_sound 1.2 (libspeex)<sup>†</sup>
+| **[AIFF]**            | No             | SDL\_sound 1.2 (internal)<sup>†</sup>
+| **[AU]**              | No             | SDL\_sound 1.2 (internal)<sup>†</sup>
+| **[VOC]**             | No             | SDL\_sound 1.2 (internal)<sup>†</sup>
+
```

Comment:
This is a nice table!  

Did some testing this afternoon comparing latest builds of staging-master and SVN-r4296.

Staging:
 - Supports: Opus, FLAC, Vorbis, MP3, WAV, and SonyWave64 (and nothing else)
 - Handles tracks encoded at 22, 44, and 48 kHz in either mono or stereo
 - Calculates accurate track MSF positions

SVN:
 - Wiki [mentions](https://www.dosbox.com/wiki/MOUNT#Mounting_a_CUE.2FBIN-Pair_as_volume) supporting Vorbis and MP3, but MP3 is broken. Testing shows I can load AIFF, AU, WAV, and Vorbis. FLAC, MP3, Speex, and VOC are broken (despite in theory SDL_Sound 1.2 saying these are supported).
 - Handled tracks exclusively encoded at 44 kHz stereo
 - Calculates accurate track MSF positions for BIN/CUE only (not file-based tracks)

I'm not sure how to represent it nicely (table? or another footer?), but it's worth showing that `staging` supports the varying sampling rates is stereo or mono, and accurate MSFs versus SVN.  

It's probably fair to drop Speex, AIFF, AU, and VOC formats from the table entirely, given they were never really supported and I doubt people are interested in them.

Reply:
For encoding voice, Xiph considers Vorbis no optimal and [recommends Speex instead](https://xiph.org/vorbis/faq/#speech); which itself is now deprecated in favor of Opus. 

For those encoding CD-DA that's primarily composed for speech, Opus or FLAC should be prefered.

Reply:
I think I'll keep AIFF (because it is explicitly "whitelisted" as supported in SVN code). TIL .W64 aka SonyWave64 exists ;) (it doesn't even have a wiki article) - I'm not sure if we want to advertise this support?

Regarding mp3: it is completely broken in DOSBox 0.74-3 in my experience as well. But it's more likely to be a fault of SDL_sound or libmpg123 and specific to my Linux distro (not sure). I had Boxtron users reporting mp3 playback working ok… but I think they might've used DOSBox-ECE or one of the forks provided by Arch (and use your patch unknowingly).

I'll reorganize the table a bit and add more info you mentioned.